docs/managed-datahub/approval-workflows.md

```diff
@@ -0,0 +1,184 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Approval Workflows
+
+<FeatureAvailability saasOnly />
+
+## Overview
+
+Keeping all your metadata properly classified can be hard work when you only have a limited number of trusted data stewards. With Acryl, you can source proposals of tags and glossary terms associated to datasets or dataset schema fields. These proposals may come from users with limited context or programatic processes using hueristics. Then, data stewards and data owners can go through them and only approve proposals they consider correct. This reduces the burden of your stewards and owners while increasing coverage.
```

Comment:
Should we do `With Managed DataHub` instead of `With Acryl`? Our Feature Availability component calls it the former

Reply:
Prop case Tags, Glossary Terms, Dataset, Dataset Schema

Reply:
addressing!